dockerRunner.go

```diff
@@ -774,32 +804,41 @@ func (dr *dockerRunnerImpl) StopContainers() {
 
 		wg.Wait()
 
-		log.Info().Msgf("Stopped %v containers", len(dr.runningContainerIDs))
+		log.Info().Msgf("Stopped %v containers", len(containerIDs))
 	} else {
 		log.Info().Msg("No containers to stop")
 	}
 }
 
-func (dr *dockerRunnerImpl) addRunningContainerID(containerID string) {
+func (dr *dockerRunnerImpl) StopAllContainers() {
+
+	allRunningContainerIDs := append(dr.runningStageContainerIDs, dr.runningSingleStageServiceContainerIDs...)
```

Comment:
Unfortunately you don't seem to be able to concat multiple arrays at once, perhaps a helper would make this more readable and less prone to errors.